src/Fable.Transforms/Python/Fable2Python.fs

```diff
@@ -546,7 +546,7 @@ module Helpers =
             | ""
             | "." -> fileDir
             | _ ->
-                IO.Path.GetFullPath(Path.Combine(Path.GetDirectoryName(com.CurrentFile), fileDir))
+                Path.GetFullPath(Path.Combine(Path.GetDirectoryName(com.CurrentFile), fileDir))
```

Comment:
I usually like to prefix with `IO.` to make sure we're not using the internal Fable polyfills but probably we should resolve the potential confusion in a better way (if Fable's polyfill doesn't have the exact same behavior we should change the name).

Reply:
@alfonsogarciacaro In this case we do want the Fable prelude version of it, which compile to .NET `Sytem.IO.Path.GetFullPath` when not compiled with Fable.